README.md

```diff
@@ -9,3 +9,20 @@ Healer is an emerging nonprofit technology initiative facilitating the efforts o
 
 ## Where can I get more information?
 This document is pretty much it for right now. If you'd like to get in touch with a human, [look here](https://github.com/sisk).
+
+# Development
+## Requirements
+* Ruby 2.5.3
+* Bundler
+* Docker
+
+## Getting Started
+* `cp .env.example .env`
+* `bundle install`
+* `docker-compose -f docker-compose.dev.yml up`
+* `bin/rake db:create:all`
+* `bin/rake db:migrate:all`
```

Comment:
`db:migrate:all` is not a valid command – should be `db:migrate`

Reply:
Copy pasta :)